### PR TITLE
Add simple test example

### PR DESCRIPTION
**Pull request by kimikowang**

### CODE REVIEW

File: pkg/test/cluster/app.go [hunk 1]
```diff
@@ -170,6 +170,11 @@ func (a *app) Call(url string, count int, headers http.Header) (test.AppCallResu
 	return out, nil
 }
 
+// CallFortio implements the test.DeployedApp interface
+func (a *app) CallFortio(cmd string) (test.AppCallFortioResponse, error) {
```
Comment (nmittler) [inline pkg/test/cluster/app.go]: I think we should keep this a separate object altogether for now.  Let's make a `FortioApp` (or something like that)

Comment (ozevren) [reply pkg/test/cluster/app.go]: +1.

**Comment by nmittler** [comment]
@kimikowang why did you close this?

**Comment by kimikowang** [comment]
Just tried to reimplement the cases... Will create a new PR soon. @nmittler

**Pull request closed without merging**